watertap/property_models/water_prop_pack.py

```diff
@@ -906,7 +908,7 @@ def _pressure_sat(self):
             doc="Saturation vapor pressure",
         )
 
-        # vapor pressure, eq. 5 and 6 in Nayar et al.(2016)
+        # Nayar et al.(2016), eq. 5 and 6, 0-180 C
         def rule_pressure_sat(b):
```

Comment:
I suggest adding saturation temperature and saturation enthalpy to the water property package, alongside the existing saturation pressure (not necessarily in this PR). These additions would be beneficial for modeling heating systems

Reply:
I'd like to add that IDAES does have the IAPWS prop pack (https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/property_models/iapws95.html?highlight=iapws) which might have additional properties that would be useful.